diesel/src/pg/types/date_and_time/chrono.rs

```diff
@@ -64,6 +64,13 @@ impl FromSql<Timestamptz, Pg> for DateTime<Utc> {
     }
 }
 
+impl FromSql<Timestamptz, Pg> for DateTime<Local> {
+    fn from_sql(bytes: Option<&[u8]>) -> deserialize::Result<Self> {
+        let naive_date_time = <NaiveDateTime as FromSql<Timestamptz, Pg>>::from_sql(bytes)?;
+        Ok(DateTime::from_utc(naive_date_time, *Local::now().offset()))
```

Comment:
What do you think about this instead? `Local::from_utc_datetime(&naive_date_time)`

Reply:
sure, why not? Change to use `Local::from_utc_datetime` pushed.